pipelines/run-pipelines-on-kubeflow-pipelines/README.md

```diff
@@ -0,0 +1,302 @@
+<!--
+{% comment %}
+Copyright 2021-2021 Elyra Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+## Run pipelines on Kubeflow Pipelines
+
+A [pipeline](https://elyra.readthedocs.io/en/stable/user_guide/pipelines.html) comprises one or more nodes that are (in many cases) connected to define execution dependencies. Each node is implemented by a [component](https://elyra.readthedocs.io/en/stable/user_guide/pipeline-components.html) and typically performs only a single task, such as loading data, processing data, training a model, or sending an email.
+
+A _generic pipeline_ comprises nodes that are implemented using _generic components_ Elyra includes generic components that run Jupyter notebooks, Python scripts, and R scripts. Generic components have in common that they are supported in every Elyra pipelines runtime environment: local/JupyterLab, Kubeflow Pipelines, and Apache Airflow.
+
+The following tutorials cover generic pipelines:
+- [Introduction to generic pipelines](../introduction-to-generic-pipelines)
+- [Run generic pipelines on Kubeflow Pipelines](../run-generic-pipelines-on-kubeflow-pipelines)
+- [Run generic pipelines on Apache Airflow](../run-generic-pipelines-on-apache-airflow)
+
+A _runtime specific_ pipeline comprises nodes that are implemented using generic components or _custom components_. Custom components are runtime specific and user-provided.
+
+In this intermediate tutorial you will learn how to add [Kubeflow Pipelines components](https://www.kubeflow.org/docs/components/pipelines/sdk/component-development/) to Elyra and how to utilize them in pipelines.
+
+![The completed tutorial pipeline](doc/images/completed-tutorial-pipeline.png)
+
+The features described in this tutorial require Elyra v3.2.0. The tutorial instructions were last updated using Elyra v3.2.0 and Kubeflow v1.3.0.
+
+> Elyra v.3.2.0 does not support [Kubeflow Pipelines Python function-based components](https://www.kubeflow.org/docs/components/pipelines/sdk/python-function-components/).
+
+### Prerequisites
+
+- [JupyterLab 3.x with the Elyra extension v3.x (or newer) installed](https://elyra.readthedocs.io/en/stable/getting_started/installation.html).
+- Access to a [local](https://elyra.readthedocs.io/en/stable/recipes/deploying-kubeflow-locally-for-dev.html) or [cloud](https://www.kubeflow.org/docs/started/cloud/) Kubeflow Pipelines deployment.
```

Comment:
The `cloud` link is broken here and I'm not sure there's a great substitute right now. Maybe this one: https://www.kubeflow.org/docs/started/installing-kubeflow/